astropy/utils/decorators.py

```diff
@@ -798,14 +790,7 @@ def __get__(self, obj, objtype=None):
             mcls = type(objtype)
             clsmeth = getattr(mcls, self.__func__.__name__, None)
             if callable(clsmeth):
-                if isinstance(clsmeth, types.MethodType):
```

Comment:
Not sure if that's applicable here but aren't bound methods still `MethodType`s? Not sure if it's possible or useful to decorator one though.

Reply:
but in the python3 case wouldn't this just be the same, so no need to do the conditional?